frontend/src/components/history-page/sort-button/sort-button.tsx

```diff
@@ -64,13 +57,17 @@ const toggleDirection = (direction: SortModeEnum) => {
  * @param onDirectionChange Callback that is fired when the sorting direction is changed
  * @param direction The sorting direction that is used
  */
-export const SortButton: React.FC<SortButtonProps> = ({ children, variant, onDirectionChange, direction }) => {
+export const SortButton: React.FC<SortButtonProps> = ({ children, onDirectionChange, direction }) => {
   const toggleSort = () => {
     onDirectionChange(toggleDirection(direction))
   }
 
   return (
-    <IconButton onClick={toggleSort} variant={variant} icon={getIcon(direction)} border={true}>
+    <IconButton
+      onClick={toggleSort}
+      variant={direction === SortModeEnum.no ? 'outline-light' : 'light'}
+      icon={getIcon(direction)}
```

Comment:
This is a function. Does React handle it properly or should we encapsulate it with a useMemo?

Reply:
not necessary.